ci-operator/step-registry/hypershift/mce/power/create/ibmcloud-workers/infra/hypershift-mce-power-create-ibmcloud-workers-infra-commands.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+set -x
+
+# Agent hosted cluster configs
+HOSTED_CLUSTER_NAME="$(printf $PROW_JOB_ID|sha256sum|cut -c-20)"
+export HOSTED_CLUSTER_NAME
```

Comment:
These 2 steps of lines 6-7 ca be done in just one:

```
export HOSTED_CLUSTER_NAME="$(printf $PROW_JOB_ID|sha256sum|cut -c-20)"
```

Reply:
Changed it, Thanks!